public/index.css

```diff
@@ -22,6 +22,38 @@ body {
   padding: 0 3%;
 }
 
+.faq-container {
+  display: flex;
+  flex-direction: column;
+  padding: 0 25%;
```

Comment:
The 25% padding here is a bit too much for mobile view. Together with the `faq-item` padding, the accordion shrinks to show 1-2 words on each line. We should think of a better solution. Maybe we can center the entire accordion and give it a minimum width?